modules/lightning_features/lightning_media/modules/lightning_media_image/lightning_media_image.install

```diff
@@ -562,18 +564,14 @@ function lightning_media_image_update_8003() {
 }
 
 /**
- * Grants image browser access to media_creator role and creator content role.
+ * Removed in Lightning 8.x-2.07.
+ *
+ * Formerly added the 'access image_browser entity browser pages' permission to
+ * the media_creator, media_manager and creator content roles.
```

Comment:
It's not clear from Lightning's API, but this is misphrased -- media_creator and media_manager are actual user roles that ship with Lightning Media, whereas "creator" is a meta-role that is managed by the lightning.content_roles service. So this should say "...permission to the media_creator and media_manager roles, as well as the creator content role."